Python/Product/Analysis/PythonKeywords.cs

```diff
@@ -72,15 +72,15 @@ public static class PythonKeywords {
         public static IEnumerable<string> Expression(PythonLanguageVersion version = PythonLanguageVersion.None) {
             yield return "and";
             yield return "as";
-            if (version >= PythonLanguageVersion.V35) {
+            if (version == PythonLanguageVersion.None || version >= PythonLanguageVersion.V35) {

```

Comment:
Yes,
static bool IsNone(this PythonLanguageVersion version){
  return version == PythonLanguageVersion.None;
}